scripts/tests/lnv2-module-test.sh

```diff
@@ -10,3 +10,5 @@ add_target_dir_to_path
 
 gateway_registration_sanity
 self_payment
+lightning_payment --gw-type lnd
```

Comment:
This can be split into a different script and run in parallel in CI if we think this will make the test too slow

Reply:
no longer necessary